TidepoolOnboardingKitUI/View Controllers/PrescriptionReviewUICoordinator.swift

```diff
@@ -273,7 +273,10 @@ class PrescriptionReviewUICoordinator: UINavigationController, OnboardingNotifyi
     }
     
     private func hostingController<Content: View>(rootView: Content) -> DismissibleHostingController {
-        return DismissibleHostingController(rootView: rootView.environment(\.appName, Bundle.main.bundleDisplayName), colorPalette: colorPalette)
+        return DismissibleHostingController(rootView: rootView
+                                                .environment(\.appName, Bundle.main.bundleDisplayName)
+                                                .environmentObject(displayGlucoseUnitObservable),
```

Comment:
[nit] My preference is for adding environment objects first and then environment values, but that's just me.